torch/_inductor/codecache.py

```diff
@@ -407,7 +407,7 @@ def get_path(
 def get_hash(
     content: Union[str, bytes], extra: str = "", hash_type: str = "code"
 ) -> str:
-    if hash_type in {"amdgcn", "code", "ptx"}:
+    if hash_type in {"amdgcn", "code", "ptx", "spv"}:
         return code_hash(content, extra)
     if hash_type in {"cubin", "hsaco", "spv"}:
```

Comment:
It probably doesn't matter in terms of functionality, but "spv" will not hit this branch anymore.